File: _Internal/Model_NorthwindIB_NH/Entities/Supplier.cs
namespace Models.NorthwindIB.NH
{
    using System;
    using System.Collections.Generic;

    public partial class Supplier
    {
        private Location _location;
        public Supplier()
        {
            this.Products = new HashSet<Product>();
        }

        public virtual int SupplierID { get; set; }
        public virtual string CompanyName { get; set; }
        public virtual string ContactName { get; set; }
        public virtual string ContactTitle { get; set; }
        public virtual string Phone { get; set; }
        public virtual string Fax { get; set; }
        public virtual string HomePage { get; set; }
        public virtual int RowVersion { get; set; }

        public virtual Location Location
        {
            get { return _location ?? (_location = new Location()); }
            set { _location = value; }
        }

        public virtual ICollection<Product> Products { get; set; }
    }
}

File: Samples_Unpublished/Java/NorthBreeze/WEB-INF/src/northwind/model/Supplier.java
package northwind.model;
import java.util.Set;

public class Supplier {
    private int supplierID;
    private String companyName;
    private String contactName;
    private String contactTitle;
    private String phone;
    private String fax;
    private String homePage;
    private int rowVersion;

    private Location location;

    private Set<Product> products;

	public int getSupplierID() {
		return supplierID;
	}

	public void setSupplierID(int supplierID) {
		this.supplierID = supplierID;
	}

	public String getCompanyName() {
		return companyName;
	}

	public void setCompanyName(String companyName) {
		this.companyName = companyName;
	}

	public String getContactName() {
		return contactName;
	}

	public void setContactName(String contactName) {
		this.contactName = contactName;
	}

	public String getContactTitle() {
		return contactTitle;
	}

	public void setContactTitle(String contactTitle) {
		this.contactTitle = contactTitle;
	}

	public String getPhone() {
		return phone;
	}

	public void setPhone(String phone) {
		this.phone = phone;
	}

	public String getFax() {
		return fax;
	}

	public void setFax(String fax) {
		this.fax = fax;
	}

	public String getHomePage() {
		return homePage;
	}

	public void setHomePage(String homePage) {
		this.homePage = homePage;
	}

	public int getRowVersion() {
		return rowVersion;
	}

	public void setRowVersion(int rowVersion) {
		this.rowVersion = rowVersion;
	}

	public Location getLocation() {
		return location;
	}

	public void setLocation(Location location) {
		this.location = location;
	}

	public Set<Product> getProducts() {
		return products;
	}

	public void setProducts(Set<Product> products) {
		this.products = products;
	}

}
